src/main/java/com/example/sinitto/auth/controller/AuthControllerAdvice.java

```diff
@@ -0,0 +1,59 @@
+package com.example.sinitto.auth.controller;
+
+import com.example.sinitto.auth.exception.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.net.URI;
+
+@RestControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice(basePackages = "com.example.sinitto.auth")
````
로 해주셔야 `auth` 도메인관련 예외만 처리가 가능합니다. `basePackages`가 없으면 애플리케이션 전역을 커버한다고 하네요! 예상치 못한 문제가 생길 수 도 있으니 예외처리 범위를 설정하는건 어떻게 생각하실지.

Reply:
몰랐던 내용이네요. 정보 감사드립니다. 적용했습니다 :)